k8s/crds.yaml

```diff
@@ -1688,6 +1688,95 @@ spec:
                 type: integer
               currentState:
                 type: string
+<<<<<<< HEAD
+=======
+              instanceEngines:
+                additionalProperties:
```

Comment:
Looks like 1.4.x doesn't have this `instanceEngines` field as we haven't combined the instance manager